protocol-units/execution/opt-executor/src/executor/execution.rs

```diff
@@ -260,6 +265,24 @@ mod tests {
 		SignedTransaction::new(raw_transaction, public_key, Ed25519Signature::dummy_signature())
 	}
 
+	fn create_signed_transaction_for_account(
```

Comment:
This seems like the only significant chunk of code that should be performing validation, but I'm not exactly seeing gas fees for validation 🤔